src/serverMain/kotlin/com/studo/campusqr/Server.kt

```diff
@@ -103,6 +103,7 @@ suspend fun main() {
 
         route("{id}") {
           post("visit") { call.visitLocation() }
+          post("guestVisit") { call.getAuthenticatedCall()?.guestVisitLocation() }
```

Comment:
is "guestVisit" meant to be "guestCheckIn"?

Reply:
if yes, this should probably not use the `getAuthenticatedCall()`